includes/cenchat-settings.php

```diff
@@ -0,0 +1,95 @@
+<?php
+class Cenchat_Settings {
+
+    public function create_menu() {
+        add_menu_page( 
+            'Cenchat', 
+            'Cenchat', 
+            'manage_options', 
+            'cenchat', 
+            array( 
+                $this, 
+                'output_settings_page' 
+            ) 
+        );
+    }
+
+    public function init_settings() {
+        register_setting( 
+            'cenchat', 
+            'cenchat_id', 
+            array( 
+                $this, 
+                'validate_input' 
+            ) 
+        );
+        add_settings_section( 
+            'cenchat_placeholder', 
+            '', 
+            array( 
+                $this, 
+                'section_placeholder' 
+            ), 
+            'cenchat'
+        );
+        add_settings_field( 
+            'cenchat_id_text', 
+            'Cenchat ID', 
+            array( 
+                $this, 
+                'text_input' 
+            ), 
+            'cenchat', 
+            'cenchat_placeholder' 
+        );
+    }
+
+    public function validate_input( $input ) {
+        $input['id_text'] = wp_filter_nohtml_kses( $input['id_text'] );	
+        return $input;
+    }
+
+    public function section_placeholder() {
+        // Do Nothing
+    }
+
+    public function text_input() {
+        $options = get_option( 'cenchat_id' );
+        echo "<input id='cenchat_id_text' name='cenchat_id[id_text]' size='40' type='text' value='{$options['id_text']}'>";
+    }
+
+    public function output_settings_page() {
+        if ( ! current_user_can( 'manage_options' ) ) {
+            return;
+        }
+
+        if ( isset( $_GET['settings-updated'] ) ) {
+            add_settings_error( 
+                'cenchat_messages', 
+                'cenchat_message', 
+                __( 'Saved', 'cenchat' ), 
+                'updated' 
+            );
+        }
+        settings_errors( 'cenchat_messages' );
+        ?>
+        <div class="wrap">
```

Comment:
Can we move this to `admin/partials` folder? I think a regular function that accepts the necessary params would suffice.